src/main/java/com/google/sps/data/User.java

```diff
@@ -3,14 +3,25 @@
 import java.util.Collection;
 import java.util.HashSet;
 
+
 public final class User {
+  public static enum profileType {
+    STUDENT, MENTOR;
+  };
 
-  private final String name;
-  private final Collection<String> productArea;
+  private Long id;
+  private String company;
+  private String major;
+  private String name;
+  private String occupation;
+  private String profileInput;
```

Comment:
what does profileInput mean?

Reply:
It was an old data member, I removed it. Done